crates/extensions/tedge_gen_mapper/pipelines/te_to_c8y.js

```diff
@@ -0,0 +1,132 @@
+/// Transform:
+///
+/// ```
+/// [te/device/main///m/example] {
+///     "time": "2020-10-15T05:30:47+00:00",
+///     "temperature": 25,
+///     "location": {
+///         "latitude": 32.54,
+///         "longitude": -117.67,
+///         "altitude": 98.6
+///     },
+///     "pressure": 98
+/// }
+/// ```
+///
+/// into
+///
+/// ```
+/// [c8y/measurement/measurements/create] {
+///     "time": "2020-10-15T05:30:47Z",
+///     "type": "example",
+///     "temperature": {
+///         "temperature": {
+///             "value": 25
+///         }
+///     },
+///     "location": {
+///         "latitude": {
+///             "value": 32.54
+///         },
+///         "longitude": {
+///             "value": -117.67
+///         },
+///         "altitude": {
+///             "value": 98.6
+///         }
+///     },
+///     "pressure": {
+///         "pressure": {
+///             "value": 98
+///         }
+///     }
+/// }
```

Comment:
As discussed offline, we still need to figure out a way to provide the external id of the entities to this `process` function.